src/MicroserviceApiClient.js

```diff
@@ -4,7 +4,7 @@ const url = require('url');
 module.exports = class MicroserviceApiClient {
 
     constructor(baseUrl, {requestLibrary = superagent} = {}) {
-        this.base = baseUrl;
+        this.base = this._normalizeBase(baseUrl);
```

Comment:
Is there a reason to flag this function as internal in think it could be handy in the tests in some cases.

Reply:
Nope, i just didn't see any use case. But we can easily make it public.